startupscript/butane/003-create-docker-network.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# create-docker-network.sh creates a docker network app-network if it does not exist.
```

Comment:
Can you add a bit more about why it the network is created? What is it used for?